### DISCUSSION TITLE
Shortcuts on job page

### DISCUSSION THREAD

**Discussion by mamachanko** [concourse/concourse#6124] [Ideas]
## Summary

The shortcuts on the web page are a great help, particularly `?` to show them and `T` to trigger a build. However, they don't work on a build page when there have been no builds yet.
![Screenshot 2020-10-06 at 15 11 41](https://user-images.githubusercontent.com/1182927/95206137-9cd4db00-07e6-11eb-863b-b5c61037b24b.png)

## Steps to reproduce

 * Spin up a pristine pipeline with a job
 * Go to its build web page
 * Hit `?` or `T`

## Expected results

* `?` should show shortcuts
* `T` should trigger a build
* etc. 

## Actual results

* nothing happens

## Triaging info

* Concourse version: 5.8.1
* Browser (if applicable): Chrome
* Did this used to work?: I don't know.

**Comment by jamieklassen** [comment]
Converting this to a discussion, because it's actually a feature request! That "build page with no builds yet" is actually called the "job page" - you can see the URL path is `/teams/:team/pipelines/:pipeline/jobs/:job` with no `/builds/:build` at the end. You can visit this page once there are builds, too - by clicking the job name in the top banner when you are on the build page.

The build page has a number of shortcuts (moving backwards and forwards in the build history, scrolling the logs, aborting a running build or rerunning a build) which don't really make sense on the job page. However a keyboard shortcut for triggering a build _would_ make sense on the job page! Maybe a shortcut for pausing/unpausing the job, too.

**Reply by mamachanko** [reply]
@jamieklassen thank you for kindly turning this into a feature request. Much appreciated. Thanks for pointing out that we're actually talking about the job page. Yes, I think it would be helpful to be able to trigger a build from said page.

If there are not available shortcuts, it would helpful if the shortcut popup would say that. As in "Howdy, here I am as you expected, yet I bring unfortunate news: there are no shortcuts on this page".
